ciri/conf/getopt.c

```diff
@@ -0,0 +1,31 @@
+#include <stdlib.h>
+
+#include "ciri/conf/provider.h"
+#include "ciri/conf/config_args.h"
+
+retcode_t ciri_conf_init(ciri_config_t* config) {
+  if (config == NULL) {
+      return 1;
+  }
+  config->tcp_port = 14260;
+  config->udp_port = 14261;
```

Comment:
Default values shouldn't be in providers.
You can #define them outside and use them here.

Reply:
How about #define in `config_args.h`?

Reply:
I would prefer another file with just default values

Reply:
Sure!